okapi-core/src/main/java/org/folio/okapi/bean/ModuleUser.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.okapi.bean;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Tells how a module is to be deployed. Either by exec'ing a command (and
+ * killing the process afterwards), or by using command lines to start and stop
+ * it.
```

Comment:
Copy & paste from https://github.com/folio-org/okapi/blob/v4.12.0/okapi-core/src/main/java/org/folio/okapi/bean/LaunchDescriptor.java , needs changes for ModuleUser.

Reply:
done